src/manage_nsfs/manage_nsfs_validations.js

```diff
@@ -637,6 +641,37 @@ function validate_whitelist_ips(ips_to_validate) {
     }
 }
 
+///////////////////////////////////
+////  CONNECTION VALIDATIONS   ////
+///////////////////////////////////
+
+function validate_connection_args(user_input, action) {
+
+    //name is mandatory for all except LIST and ADD
```

Comment:
I don't understand this comment - why name is not mandatory for "connection add" since I saw:

> Usage:
>     connection add [flags]
> Flags:
>     --name <string>                                                         Set the name for the connection